integrationTests/src/test/java/org/sonarqube/gradle/GradleTest.java

```diff
@@ -269,20 +270,19 @@ public void testKotlinMultiplatformProject() throws Exception {
 
     assertThat(binaries).containsExactlyInAnyOrder(
       baseDir.resolve("build/classes/kotlin/jvm/main").toString(),
-      baseDir.resolve("build/classes/java/main").toString()
-    );
+      baseDir.resolve("build/classes/java/main").toString());
 
     assertThat(sources).containsExactlyInAnyOrder(
       baseDir.resolve("src/commonMain/kotlin").toString(),
       baseDir.resolve("src/jvmMain/kotlin").toString(),
-      baseDir.resolve("src/jvmMain/java").toString()
-    );
+      baseDir.resolve("src/jvmMain/java").toString());
   }
 
   @Test
   public void testKotlinMultiplatformWithSubmoduleProject() throws Exception {
     Assume.assumeTrue("Tests only applies to version 6.8.3 or greater", getGradleVersion().isGreaterThanOrEqualTo("6.8.3"));
-    Properties props = runGradlewSonarSimulationModeWithEnv("/kotlin-multiplatform-with-submodule", emptyMap(), "compileCommonMainKotlinMetadata", "compileKotlinJvm", "compileKotlinMetadata", "compileTestKotlinJvm");
+    Properties props = runGradlewSonarSimulationModeWithEnv("/kotlin-multiplatform-with-submodule", emptyMap(), "compileCommonMainKotlinMetadata", "compileKotlinJvm",
+      "compileKotlinMetadata", "compileTestKotlinJvm");
```

Comment:
Let's revert this change
```suggestion
    Properties props = runGradlewSonarSimulationModeWithEnv("/kotlin-multiplatform-with-submodule", emptyMap(), "compileCommonMainKotlinMetadata", "compileKotlinJvm", "compileKotlinMetadata", "compileTestKotlinJvm");
```